### ISSUE TITLE
ProfileNotFound: The config profile (dev) could not be found

### ISSUE THREAD

**Issue by markelrod** [castlabs/electron-releases#95]
When trying to sign up for an account we get a ProfileNotFound error.

Here is an example with dummy data in place of real values:

```# python3.7 -m castlabs_evs.account signup
Signing up for castLabs EVS
 - A valid e-mail address is required for account verification
>> E-mail Address [person@example.com]: 
>> First Name [Firstname]: 
>> Last Name [Lastname]: 
>> Organization [Example Org]: 
>> Account Name [Account]: 
>> Password []: 
>> Verify Password []: 
ProfileNotFound: The config profile (dev) could not be found

**Comment by khwaaj** [comment]
Interestingly this appears to be because of the default behavior of `boto3`, which is used for account creation. If one of the environment variables `AWS_PROFILE` or `AWS_DEFAULT_PROFILE` is set to indicate a profile that does not exist in the local AWS configuration it results in `ProfileNotFound`.

You should be able to work around this by making sure none of the two environment variables mentioned above are set when you run the EVS tool. I'll see about making the EVS tools handle this scenario better in future versions, thanks for reporting it!

**Comment by markelrod** [comment]
Thanks Emil - we were using a docker container to build for win32 and did not realize it had an AWS_PROFILE set (nor that it mattered). We have it working now.

**Comment by khwaaj** [comment]
I just published `castlabs_evs-1.0.0` to PyPi as it is now mature enough to drop the release candidate status. It fixes a few minor things, including handling AWS related environment variables, so if they happen to be set it should not matter anymore.

Please try it out and unless you have anything else I think we can close this issue, no?
